sotodlib/io/bookbinder.py

```diff
@@ -171,7 +255,13 @@ def __init__(self, files, book_id, drop_duplicates=False, log=None):
             self.log = logging.getLogger('bookbinder')
         else:
             self.log = log
-    
+
+        if self.hkdata.az is None:
+            self.log.warning("No ACU data specified in hk_fields!")
```

Comment:
This needs to be a bypass-able error. If I'm reading through this setup correctly then if the mount agent changes name or if the imprinter configs are configured incorrectly then we'll just quietly get books without mount data which could take awhile to catch.